decidim-meetings/app/commands/decidim/meetings/admin/update_meeting.rb

```diff
@@ -48,7 +48,9 @@ def update_meeting!
             event_class: Decidim::Meetings::UpdateMeetingEvent,
             resource: meeting,
             recipient_ids: meeting.users_to_notify.pluck(:id),
-            user: form.current_user
+            extra: {
+              actor_id: form.current_user.id
```

Comment:
Why actor?

Reply:
Because it's the user that performs the action. Any idea for a better naming?

Reply:
updated_by_id ? Or do we need a generic way to refer to it? Why do we need the user that updated the meeting for the notification?

Reply:
We don't need it right now. I am ok removing it. @mrcasals ?

Reply:
OK to the name change, but I think having this can be good, for example we might want to change the text to show the name of the user that did the action.

Reply:
I agree but not really for this action since it would be an admin, right?

Reply:
@oriolgual @mrcasals I renamed both fields. I think this information will be useful if we want to start some kind of activity log for the admin users.

Reply:
I don't want to merge activity log with notifications @beagleknight, I know we'll need it in the future but those are two different concerns.

Reply:
Ok, I agree with that.